diff --git a/src/libexpr-c/meson.build b/src/libexpr-c/meson.build
index 9487132cf..39dd3ed63 100644
--- a/src/libexpr-c/meson.build
+++ b/src/libexpr-c/meson.build
@@ -79,7 +79,6 @@ this_library = library(
   dependencies : deps_public + deps_private + deps_other,
   include_directories : include_dirs,
   link_args: linker_export_flags,
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libexpr-test-support/meson.build b/src/libexpr-test-support/meson.build
index 56e814cd1..771c3d071 100644
--- a/src/libexpr-test-support/meson.build
+++ b/src/libexpr-test-support/meson.build
@@ -63,7 +63,6 @@ this_library = library(
   # TODO: Remove `-lrapidcheck` when https://github.com/emil-e/rapidcheck/pull/326
   #       is available. See also ../libutil/build.meson
   link_args: linker_export_flags + ['-lrapidcheck'],
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libexpr/meson.build b/src/libexpr/meson.build
index b33aebc86..e60216664 100644
--- a/src/libexpr/meson.build
+++ b/src/libexpr/meson.build
@@ -195,7 +195,6 @@ this_library = library(
   lexer_tab,
   generated_headers,
   dependencies : deps_public + deps_private + deps_other,
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libfetchers/meson.build b/src/libfetchers/meson.build
index 58afbb7d0..7066c8a62 100644
--- a/src/libfetchers/meson.build
+++ b/src/libfetchers/meson.build
@@ -80,7 +80,6 @@ this_library = library(
   'nixfetchers',
   sources,
   dependencies : deps_public + deps_private + deps_other,
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libflake/meson.build b/src/libflake/meson.build
index b757d0d76..f6663d7b2 100644
--- a/src/libflake/meson.build
+++ b/src/libflake/meson.build
@@ -62,7 +62,6 @@ this_library = library(
   'nixflake',
   sources,
   dependencies : deps_public + deps_private + deps_other,
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libstore-c/meson.build b/src/libstore-c/meson.build
index 17d18609f..711aaead2 100644
--- a/src/libstore-c/meson.build
+++ b/src/libstore-c/meson.build
@@ -71,7 +71,6 @@ this_library = library(
   dependencies : deps_public + deps_private + deps_other,
   include_directories : include_dirs,
   link_args: linker_export_flags,
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libstore-test-support/meson.build b/src/libstore-test-support/meson.build
index 59d649889..3f02b9438 100644
--- a/src/libstore-test-support/meson.build
+++ b/src/libstore-test-support/meson.build
@@ -65,7 +65,6 @@ this_library = library(
   # TODO: Remove `-lrapidcheck` when https://github.com/emil-e/rapidcheck/pull/326
   #       is available. See also ../libutil/build.meson
   link_args: linker_export_flags + ['-lrapidcheck'],
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libstore/meson.build b/src/libstore/meson.build
index 79d912497..fb427c6b9 100644
--- a/src/libstore/meson.build
+++ b/src/libstore/meson.build
@@ -425,7 +425,6 @@ this_library = library(
   include_directories : include_dirs,
   cpp_args : cpp_args,
   link_args: linker_export_flags,
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libutil-c/meson.build b/src/libutil-c/meson.build
index ac1297665..b000299f9 100644
--- a/src/libutil-c/meson.build
+++ b/src/libutil-c/meson.build
@@ -67,7 +67,6 @@ this_library = library(
   dependencies : deps_public + deps_private + deps_other,
   include_directories : include_dirs,
   link_args: linker_export_flags,
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libutil-test-support/meson.build b/src/libutil-test-support/meson.build
index db944cf06..ae52a6c6b 100644
--- a/src/libutil-test-support/meson.build
+++ b/src/libutil-test-support/meson.build
@@ -60,7 +60,6 @@ this_library = library(
   # TODO: Remove `-lrapidcheck` when https://github.com/emil-e/rapidcheck/pull/326
   #       is available. See also ../libutil/build.meson
   link_args: linker_export_flags + ['-lrapidcheck'],
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libutil/meson.build b/src/libutil/meson.build
index ac701d8fd..5a0e82ba3 100644
--- a/src/libutil/meson.build
+++ b/src/libutil/meson.build
@@ -269,7 +269,6 @@ this_library = library(
   dependencies : deps_public + deps_private + deps_other,
   include_directories : include_dirs,
   link_args: linker_export_flags,
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/perl/lib/Nix/meson.build b/src/perl/lib/Nix/meson.build
index 256e66096..9a79245cd 100644
--- a/src/perl/lib/Nix/meson.build
+++ b/src/perl/lib/Nix/meson.build
@@ -43,7 +43,6 @@ nix_perl_store_lib = library(
   'Store',
   sources : nix_perl_store_cc,
   name_prefix  : '',
-  prelink : true, # For C++ static initializers
   install : true,
   install_mode : 'rwxr-xr-x',
   install_dir : join_paths(nix_perl_install_dir, 'auto', 'Nix', 'Store'),
diff --git a/src/libcmd/meson.build b/src/libcmd/meson.build
index 4145f408a..20173f7b7 100644
--- a/src/libcmd/meson.build
+++ b/src/libcmd/meson.build
@@ -117,7 +117,6 @@ this_library = library(
   'nixcmd',
   sources,
   dependencies : deps_public + deps_private + deps_other,
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libflake-c/meson.build b/src/libflake-c/meson.build
index 85d20644d..757735a68 100644
--- a/src/libflake-c/meson.build
+++ b/src/libflake-c/meson.build
@@ -80,7 +80,6 @@ this_library = library(
   dependencies : deps_public + deps_private + deps_other,
   include_directories : include_dirs,
   link_args: linker_export_flags,
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libmain-c/meson.build b/src/libmain-c/meson.build
index d875d2c3f..e0120f364 100644
--- a/src/libmain-c/meson.build
+++ b/src/libmain-c/meson.build
@@ -72,7 +72,6 @@ this_library = library(
   dependencies : deps_public + deps_private + deps_other,
   include_directories : include_dirs,
   link_args: linker_export_flags,
-  prelink : true, # For C++ static initializers
   install : true,
 )
 
diff --git a/src/libmain/meson.build b/src/libmain/meson.build
index 00f945f49..eccf7c571 100644
--- a/src/libmain/meson.build
+++ b/src/libmain/meson.build
@@ -86,7 +86,6 @@ this_library = library(
   'nixmain',
   sources,
   dependencies : deps_public + deps_private + deps_other,
-  prelink : true, # For C++ static initializers
   install : true,
 )

